node/subsystem/src/jaeger.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Jaeger integration.
+//!
+//! See <https://www.jaegertracing.io/> for an introduction.
+//!
+//! The easiest way to try Jaeger is:
+//!
+//! - Start a docker container with the all-in-one docker image (see below).
+//! - Open your browser and navigate to <https://localhost:16686> to acces the UI.
+//!
+//! The all-in-one image can be started with:
+//!
+//! ```not_rust
+//! podman login docker.io
+//! podman run -d --name jaeger \
+//!  -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
+//!  -p 5775:5775/udp \
+//!  -p 6831:6831/udp \
+//!  -p 6832:6832/udp \
+//!  -p 5778:5778 \
+//!  -p 16686:16686 \
+//!  -p 14268:14268 \
+//!  -p 14250:14250 \
+//!  -p 9411:9411 \
+//!  jaegertracing/all-in-one:1.21
```

Comment:
```suggestion
//!  docker.io/jaegertracing/all-in-one:1.21
```

Reply:
```
[Unit]
Description=Jaeger Tracing Service
After=network.target

[Service]
User=root
ExecStartPre=-/usr/bin/podman pull jaegertracing/all-in-one:1.21
ExecStart=/usr/bin/podman run -it --rm --name jaeger \
 -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
 -p 5775:5775/udp \
 -p 6831:6831/udp \
 -p 6832:6832/udp \
 -p 5778:5778 \
 -p 16686:16686 \
 -p 14268:14268 \
 -p 14250:14250 \
 -p 9411:9411 \
 jaegertracing/all-in-one:1.21

[Install]
WantedBy=multi-user.target

```
has been running smoothly for the last couple of days.

Reply:
running the pre start command:
```
$ /usr/bin/podman pull jaegertracing/all-in-one:1.21
Completed short name "jaegertracing/all-in-one" with unqualified-search registries (origin: /etc/containers/registries.conf)
Trying to pull registry.fedoraproject.org/jaegertracing/all-in-one:1.21...
  manifest unknown: manifest unknown
Trying to pull registry.access.redhat.com/jaegertracing/all-in-one:1.21...
  name unknown: Repo not found
Trying to pull registry.centos.org/jaegertracing/all-in-one:1.21...
  manifest unknown: manifest unknown
Trying to pull docker.io/jaegertracing/all-in-one:1.21...
Getting image source signatures
Copying blob 188c0c94c7c5 done  
Copying blob 87901b057fcd done  
Copying blob 8d7d3077ebd1 done  
Copying blob 6130f8acbe29 done  
Copying config 87b921624a done  
Writing manifest to image destination
Storing signatures
<snip>
```
worked just fine but I also had an entry for docker in `/etc/containers/registries*`.

The recommended change works for me and should solid.